features/networking/pod.feature

```diff
@@ -190,3 +190,119 @@ Feature: Pod related networking scenarios
     Then the step should succeed
     And the output should not contain "ingress_policing_rate: 1953"
 
+  # @auther anusaxen@redhat.com
+  # @case_id OCP-23890
+  @admin
+  Scenario: A pod with or without hostnetwork cannot access the MCS port 22623 or 22624 on the master
+    Given I use the first master host
+    #Step to obtain master IP
+    And I run commands on the host:
+      | hostname -i |
+    Then the step should succeed
+    And evaluation of `@result[:response].strip` is stored in the :master_ip clipboard
+    
+    Given I select a random node's host
+    Given I have a project
+    #pod-for-ping will be a non-hostnetwork pod
+    And SCC "privileged" is added to the "system:serviceaccounts:<%= project.name %>" group
+    And I have a pod-for-ping in the project
+    And the pod named "hello-pod" becomes ready
+
+    When I execute on the pod:
+      | curl | -I | https://<%= cb.master_ip %>:22623/master/config | -k |
+    Then the output should contain "Connection refused"
+    When I execute on the pod:
+      | curl | -I | https://<%= cb.master_ip %>:22624/master/config | -k |
+    Then the output should contain "Connection refused"
+    
+    #hostnetwork-pod will be a hostnetwork pod
+    When I run the :create admin command with:
```

Comment:
this project had privileged, so no need to use `admin`, should be 
`When I run the :create client command with:`

Reply:
Agree, admin can be removed. I believe when we don't add privilege to project then the admin is needed

Reply:
@anuragthehatter seems this did not be updated yet.